### DISCUSSION TITLE
Can WASM builds of Altair be adopted within the CI part of Github Action?

### DISCUSSION THREAD

**Discussion by mattijn** [vega/altair#3578] [Ideas]
There is quarto-live that supports Altair charts, see https://r-wasm.github.io/quarto-live/other/py_output.html. It might be possible to setup a test that output webpages. 

Maybe similar to what was done in the Vega-Lite repository: https://github.com/vega/vega-lite/pull/9294.
Even though it might technical be possible, a perfectly fine outcome of this discussion might also be that our current tests are already sufficient enough.

**Comment by joelostblom** [comment]
I'm playing around with quarto live in a viz textbook I'm working on; you can see an early preview here https://joelostblom.github.io/viz-oer, including a couple of altair exercises. (I might reach out for feedback at some point, but if anyone would benefit from being involved in some other capacity let me know. @ChristopherDavisUCI I think we talked about this at some point).

As part of that project I'm also trying to work on https://github.com/vega/altair/issues/3434 for Altair, which could give us deployment previews and rendered diffs of the docs for each PR.
